<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>message</class>
 <widget class="QWidget" name="message">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>456</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>Copy message text</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="message_container" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="user_name">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mood_emoji">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="visible">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="play_audio_button">
          <property name="toolTip">
           <string>Play message audio</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../../../gui/resources/feather.qrc">
            <normaloff>:/light/icons/feather/light/play.svg</normaloff>:/light/icons/feather/light/play.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="copy_button">
          <property name="toolTip">
           <string>Copy message</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../../../gui/resources/feather.qrc">
            <normaloff>:/light/icons/feather/light/copy.svg</normaloff>:/light/icons/feather/light/copy.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_button">
          <property name="toolTip">
           <string>Delete message</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../../../gui/resources/feather.qrc">
            <normaloff>:/light/icons/feather/light/x-circle.svg</normaloff>:/light/icons/feather/light/x-circle.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="content_container">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="image_content">
        <property name="visible">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../gui/resources/feather.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>delete_button</sender>
   <signal>clicked()</signal>
   <receiver>message</receiver>
   <slot>delete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copy_button</sender>
   <signal>clicked()</signal>
   <receiver>message</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>-2</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>delete()</slot>
  <slot>copy()</slot>
 </slots>
</ui>
